app/models/period.rb

```diff
@@ -45,6 +45,22 @@ def to_date
     value.to_date
   end
 
+  def start_date
+    if quarter?
+      Quarter.for_date(to_date).start_date
```

Comment:
`to_quarter_period.start_date`

Reply:
oh I see, we really don't need to create a quarter here...ya.

Reply:
oh wait, I think I see a way we can clean this up a little.  I'll push up some commits.